netbout-web/src/main/java/com/netbout/dynamo/DyBout.java

```diff
@@ -121,6 +121,30 @@ public void rename(final String text) throws IOException {
     }
 
     @Override
+    @SuppressWarnings("PMD.OnlyOneReturn")
```

Comment:
@erimerturk let's remove PMD suppression here
we can use  `( ? : ):` operator. isn't it ?
or create a variable for the return value